code/modules/admin/view_variables/view_variables.dm

```diff
@@ -14,6 +14,8 @@ var/global/list/view_variables_no_assoc = list("verbs", "contents","screen","ima
 	if(!D)
 		return
 
+	var/static/cookieoffset = rand(1, 9999) //to force cookies to reset after the round.
```

Comment:
This is pretty terrible; you should be using game id and timing out your cookies.